content/tutorials/authentication/how-to-authenticate-react.md

```diff
@@ -6,3 +6,94 @@ version: v7.0
 tags: [react, getting started, authentication, client]
 description: This tutorial shows you how to authenticatie your a React app to sensenet.
 ---
+
+This tutorial shows you how to authenticatie your a React app to sensenet.
```

Comment:
is this correct? "your a React app" - (I would say "your React app" or "a React app)". if not, it is to correct also in the description